packages/connect/src/ScProvider.ts

```diff
@@ -80,7 +80,9 @@ export class ScProvider implements ProviderInterface {
     parachainSpec?: string,
     autoConnect = true,
   ) {
-    const isExtension = !!document.getElementById("substrateExtension")
+    const isExtension = !!document.getElementById(
+      "substrateConnectionExtensionAvailable",
```

Comment:
I would suggest to name this as the extension (`substrateConnectExtensionAvailable`)

Reply:
Ah yeah, that was my intention but I'm not fully awake yet apparently